documentation/GettingStarted.mdx

```diff
@@ -11,8 +10,7 @@ import { cva, cx } from 'class-variance-authority'
   <span className="text-support">with ease</span>.
 </p>
 <p className="text-center text-on-neutral-container sb-unstyled text-headline-1">
-  **Spark UI** is a React based design system focused on modularity and accessibility, using
-  TailwindCSS and Radix UI.
+  **Spark UI** is a React based design system focused on modularity and accessibility.
```

Comment:
I removed mentions of TailwindCSS and Radix, as we not only use these libraries. Also it might be an unnecessary detail, IMHO.